### PR TITLE
VACMS-2982: Remove legacy cms fields from build.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Remove fields from build that are no longer in use in the cms

## Testing done
Build

## Acceptance criteria
- [ ] Successful build

### CODE REVIEW

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 const { partialSchema } = require('../../transformers/helpers');
-const personProfileSchema = require('./node-person_profile');
```
Comment (jhonnyoddball) [inline src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js:2]: Why is this being removed?

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js [hunk 2]
```diff
@@ -120,28 +115,6 @@ module.exports = {
...
-          'fieldOffice',
-          'fieldIntroText',
-          'fieldPhotoAllowHiresDownload',
-          'fieldMedia',
-          'fieldBody',
-          'changed',
-          'entityUrl',
-        ]),
-      },
-    },
```
Comment (jhonnyoddball) [inline src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js:144]: Why is this being removed?

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js [hunk 3]
```diff
@@ -187,14 +160,9 @@ module.exports = {
...
     'fieldGovdeliveryIdNews',
     'fieldOperatingStatus',
-    'fieldFacebook',
-    'fieldFlickr',
-    'fieldInstagram',
-    'fieldTwitter',
     'fieldNicknameForThisFacility',
     'fieldLinkFacilityEmergList',
     'fieldPressReleaseBlurb',
-    'fieldLeadership',
```
Comment (jhonnyoddball) [inline src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js:197]: Why is this being removed?

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js [hunk 4]
```diff
@@ -22,15 +22,10 @@ const transform = ({
...
   fieldOperatingStatus,
-  fieldFacebook,
-  fieldFlickr,
-  fieldInstagram,
-  fieldTwitter,
   fieldNicknameForThisFacility,
   fieldRelatedLinks,
   fieldPressReleaseBlurb,
   fieldLinkFacilityEmergList,
-  fieldLeadership,
```
Comment (jhonnyoddball) [inline src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js:33]: Why is this being removed?

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js [hunk 5]
```diff
@@ -359,11 +320,6 @@ module.exports = {
     'path',
     'field_govdelivery_id_emerg',
     'field_govdelivery_id_news',
-    'field_facebook',
-    'field_flickr',
-    'field_instagram',
-    'field_twitter',
-    'field_leadership',
```
Comment (jhonnyoddball) [inline src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js:366]: Why is this being removed?

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js (hunk 1 above)
Comment (ethanteague) [reply src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js:2]: Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js (hunk 2 above)
Comment (ethanteague) [reply src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js:144]: Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh

File: src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js (hunk 3 above)
Comment (ethanteague) [reply src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js:197]: Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js (hunk 4 above)
Comment (ethanteague) [reply src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js:33]: Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js (hunk 5 above)
Comment (ethanteague) [reply src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js:366]: Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh

**Review by jhonny218 — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]
LGTM! Good cleanup

**Pull request merged**